### ISSUE TITLE
Can you release commands for training the source policies?

### ISSUE THREAD

**Issue by NagisaZj** [tianpeiyang/PTF_code#4]
Hi there! Thank you for releasing the code.

I am trying to run PTF on my own environments, and now I am trying to train source policies with your codes. It would be of great help if you can provide commands for training the source policies contained in the codebase, as I can refer to the hyper-parameter settings. 

As my environment is built upon MuJoCo and has comtinuous action spaces, I think the commands for training Reacher's source policies will help me the most. Thank you very much!

**Comment by tianpeiyang** [comment]
Hope this command will help you. {"numGames": 100000, "game": "reacher", "algorithm": "a3c", "epi_step": 99, "seed": 80, "optimizer": "adam", "run_test": false, "results_path": "results/a3c_reacher/2019-08-31_16-57-43/", "reward_decay": 0.99, "output_graph": true, "save_model": true, "summary_output_times": 10, "regular": 0.005, "learning_rate_a": 0.0001, "learning_rate_c": 0.0001, "ENTROPY_BETA": 0.0003, "N_WORKERS": 1, "USE_CPU_COUNT": true, "load_model": false, "load_model_path": "", "batch_size": 300, "update_rate": 0.4, "reward_memory": 100, "save_per_episodes": 1000, "n_layer_a_1": 256, "n_layer_a_2": 64, "n_layer_c_1": 256, "n_layer_c_2": 64, "GLOBAL_NET_SCOPE": "Global_Net", "SAVE_PATH": "model", "graph_path": "graph", "reward_output": "output", "output_filename": "out", "log": "log", "continuous_action": true, "action_clip": 1, "reward_normalize": false, "task": "t2"}
